repo/packages/H/hdfs/0/config.json

```diff
@@ -11,9 +11,25 @@
           "type": "string"
         },
         "default": [
-          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz"
+          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz",
+          "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz"
         ]
       },
+      "ld-library-path": {
```

Comment:
These values should be in alphabetical order. They seem to be in reverse alphabetical order.